waggle-dance-core/src/test/java/com/hotels/bdp/waggledance/metrics/MonitoredAspectTest.java

```diff
@@ -109,6 +115,23 @@ public void monitorSuccesses() throws Throwable {
     assertThat(rs.timer().count(), is(1L));
   }
 
+  @Test
+  public void monitorSuccessWithTags() throws Throwable {
+    aspect.monitor(pjp, monitored);
+
+    Collection<Meter> meters = meterRegistry.get("counter.Type_Anonymous.success").meters();
+    assertThat(meters.size(), is(1));
+    assertThat(((CumulativeCounter) ((ArrayList) meters).get(0)).count(), is(1.0));
```

Comment:
this double cast is very hard to read, why don't you follow the examples from the other tests?
```
 RequiredSearch rs = meterRegistry.get("counter.Type_Anonymous.success");
 assertThat(rs.counter().count(), is(1.0));
 ```